crates/sui-graphql-rpc/src/types/owner.rs

```diff
@@ -422,8 +448,35 @@ impl OwnerImpl {
         before: Option<object::Cursor>,
     ) -> Result<Connection<String, DynamicField>> {
         let page = Page::from_params(ctx.data_unchecked(), first, after, last, before)?;
-        DynamicField::paginate(ctx.data_unchecked(), page, self.0)
+        DynamicField::paginate(ctx.data_unchecked(), page, self.address)
             .await
             .extend()
     }
 }
+
+impl From<&Address> for OwnerImpl {
```

Comment:
nit: Move these into their respective modules? (It's a bit of a departure from our rule of thumb that the `trait` impl goes in the module that defines the type, but the counter to that is that you would usually impl `Into<OwnerImpl> for &Address`, but then clippy will tell you that `From` is better and it's fine to leave it where it is in the module that defines the type you're going from, because of the orphan rule.